### PR TITLE
fix: handling lang attribute in html tag

### PR DESCRIPTION
**Pull request by chijure**
handling attribute lang in tag html

### CODE REVIEW

**Comment by rschristian** [comment]
Sorry, what does this fix? Seems to all be working as-is.

**Comment by chijure** [comment]
> Sorry, what does this fix? Seems to all be working as-is.

when you first enter the language is English, and the lang = "en", but when you change the language and reload the lang is still "en"

Screen Shot 2021-07-11 at 18 16 42

File: src/lib/i18n.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ export function useLanguage() {
 		[url]
 	);
 
+	setLang(lang);
```
Comment (developit) [inline src/lib/i18n.js:22]: I believe this will cause an infinite promise loop, or at least a lot of duplicate renders.

**Comment by rschristian** [comment]
Closing in favor of #1104

Sorry for taking so long to address this, but thanks for taking a crack at it!

**Pull request closed without merging**